recipes-multimedia/gstreamer/gstreamer1.0_1.24.0.imx.bb

```diff
@@ -92,14 +92,21 @@ LIC_FILES_CHKSUM = " \
 "
 
 # Use i.MX fork of GST for customizations
-SRC_URI:remove = "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PV}.tar.xz"
+SRC_URI:remove = "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PV}.tar.xz \
+                file://0001-tests-respect-the-idententaion-used-in-meson.patch \
+                file://0002-tests-add-support-for-install-the-tests.patch \
+                file://0003-tests-use-a-dictionaries-for-environment.patch;striplevel=3 \
+                file://0004-tests-add-helper-script-to-run-the-installed_tests.patch;striplevel=3 \
+"
 SRC_URI:prepend = "${GST1.0_SRC};branch=${SRCBRANCH} "
 GST1.0_SRC ?= "gitsm://github.com/nxp-imx/gstreamer.git;protocol=https"
-SRCBRANCH = "MM_04.08.03_2312_L6.6.y"
-SRCREV = "1a43c16272a7f4274eb8260e03206a57f317d823"
+SRCBRANCH = "MM_04.09.00_2405_L6.6.y"
+SRCREV = "753c7b4471d14befd1b7c9cd3fa9af9f10ab4a08"
 
 S = "${WORKDIR}/git"
 
+PACKAGECONFIG[tests] = "-Dtests=enabled,-Dtests=disabled"
```

Comment:
This line is not needed because it is already present on line 24
https://github.com/Freescale/meta-freescale/pull/1874/commits/5c05405e1057681c698aafc3ec284500a69e1adf#diff-3a0a7153e6a185fd9204a4b9d6b987c1980cd3a2ee839fd11d72a582dbdd4675R43

Reply:
I am not sure about this, this recipe is copied from OE upstream and then these variables are replaced only at the end of the file, after the `########### i.MX overrides ################`.

The differences are:

```
PACKAGECONFIG[tests] = "-Dtests=enabled -Dinstalled_tests=true,-Dtests=disabled -Dinstalled_tests=false"
PACKAGECONFIG[tests] = "-Dtests=enabled,-Dtests=disabled"
```

So the flags are changing, but I do not know how relevant they are. That being said, should I remove this line anyway?

Thanks.

Reply:
The meson "-Dtests" doesn't make much sense without the mentioned patches above. The [PACKAGECONFIG[tests]]( https://github.com/openembedded/openembedded-core/blob/662396533177b72cc1d83e95841b27f7e42dcb20/meta/recipes-multimedia/gstreamer/gstreamer1.0_1.24.5.bb#L36) is still in use as well upstream in oe-core.

Reply:
I removed the line 

```bash
PACKAGECONFIG[tests] = "-Dtests=enabled,-Dtests=disabled"
```

thanks for clarifying.